CODEOWNERS

```diff
@@ -10,3 +10,6 @@
 
 # All .md files
 *.md @bszwarc @majakurcius @tomekpapiernik @kazydek @klaudiagrz @alexandra-simeonova @mmitoraj
+
+# The `distibution-library-image` repo
```

Comment:
That entry should go above the .md files Otherwise, TWs will be excluded from any reviews in  `distibution-library-image`.